sdk/monitor/Azure.ResourceManager.Monitor/src/autorest.md

```diff
@@ -8,12 +8,13 @@ azure-arm: true
 generate-model-factory: false
 library-name: Monitor
 namespace: Azure.ResourceManager.Monitor
-require: https://github.com/Azure/azure-rest-api-specs/blob/fce14bf881523f16b14d8ecf325e004465cdcf56/specification/monitor/resource-manager/readme.md
-tag: package-track2-stable
+require: https://github.com/Azure/azure-rest-api-specs/blob/55e5b360933b7d7467ba5a912ae4ff83806be9c7/specification/monitor/resource-manager/readme.md
+tag: package-2023-04
```

Comment:
well instead of using this tag, let us update the tag we created in this file (you will find it at the end of this file). Changing the commit id in all of the links, and adding your swagger in the list as well.
The reason that we are keeping a tag in this file directly is that dotnet SDK is now stable, and as an exception we allowed some preview swagger to participate but for the updates we will not allow more preview swaggers to join the stable version of the monitor SDK, and the tag in the swagger readme of monitor is kind of messy therefore we decide to maintain that here directly by us so that we could control which swagger is preview and which is not